prac2-pages/Siddharth_gautam_login_page/lib/other.dart

```diff
@@ -0,0 +1,210 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApplication());
```

Comment:
Please remove this. :exclamation: 

There should only be **one** entrypoint into the application, through the main() method in main.dart
- If you want to view your page, just supply the widget in the runApp() method's parameter.